pkgs/unified_analytics/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 1.1.0
 
 - Added a `okToSend` getter so that clients can easily and accurately check the state of the consent mechanism.
+- Initialize the config file with user opted out if user was opted out in legacy Flutter and Dart analytics
```

Comment:
I notice the pubspec.yaml is already bumped to 1.1.0, but it looks like that wasn't published. Is it planned to publish 1.1.0 with both these changes?

Reply:
:+1:

It's definitely a nit, but I think being disciplined about correlating pubspec.yaml version bumps with what gets actually published on pub is helpful in case we need to track later which releases have a particular bug and may need to be hotfixed.